src/components/link/link.scss

```diff
@@ -0,0 +1,97 @@
+@use '../../scss/global.scss';
+
+:host {
+  display: inline-block;
+  margin: 4px 0;
+}
+
+// base class
+.kd-link-text {
```

Comment:
Could add a transition for `color` here to smooth it out, and then also for the focus `outline`.

Reply:
didn't get it.